cgp/population.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         mutation_rate: float,
         seed: int,
         genome_params: Union[dict, List[dict]],
+        dna_init: Union[Callable[[IndividualBase], IndividualBase], None] = None,
```

Comment:
If the purpose is mainly setting the dna, why do you pass it as a Callable and not as a List?

Reply:
And I guess it would be nice to be able to set multiple initial solution for different parents. 
(or am I missing something and this is possible thanks to 'dna_init' being a callable?)

Reply:
a callable just allows for more flexibility. say you would like all outputs to initially read from the input while having an otherwise random dna. with a callable you could directly manipulate particular genes of the dna instead of having to construct it by yourself. and if you want to overwrite the whole dna you can just as easily to it.